helix-core/src/test/java/org/apache/helix/TestHelper.java

```diff
@@ -864,4 +866,36 @@ public static void printZkListeners(HelixZkClient client) throws Exception {
     }
     System.out.println("}");
   }
+
+  public static HelixZkClient createZkClient(String zkAddress) {
+    HelixZkClient.ZkClientConfig clientConfig = new HelixZkClient.ZkClientConfig()
+        .setZkSerializer(new org.apache.helix.zookeeper.datamodel.serializer.ZNRecordSerializer())
+        .setConnectInitTimeout(1000L)
+        .setOperationRetryTimeout(1000L);
```

Comment:
We do have the default value. Let's not hardcode it here.

Reply:
Yes, correct. I have removed now.